### ISSUE TITLE
Node needs error reporting when serial cable is not present

### ISSUE THREAD

**Issue by golanlevin** [yonmaor1/p5-flipdots#1]
Running the Node server at home without the device -- produces error: 
```
gl@glm1 p5-flipdots % node golive.js

serving local server at http://localhost:8081
Node.js Server Error: node:internal/process/promises:391
    triggerUncaughtException(err, true /* fromPromise */);
    ^

Error: Error: No such file or directory, cannot open /dev/tty.usbserial-A9E7G8FR
Emitted 'error' event on SerialPort instance at:
    at SerialPort._error (/Users/gl/Documents/GitHub/p5-flipdots/node_modules/@serialport/stream/dist/index.js:85:18)
    at /Users/gl/Documents/GitHub/p5-flipdots/node_modules/@serialport/stream/dist/index.js:118:18
```

**Comment by yonmaor1** [comment]
fixed in commit [ce951b3](https://github.com/yonmaor1/p5-flipdots/commit/ce951b3d63fe2924353ef2206dd9b5001339fca6)

**Comment by yonmaor1** [comment]
fixed in commit [ce951b3](https://github.com/yonmaor1/p5-flipdots/commit/ce951b3d63fe2924353ef2206dd9b5001339fca6)
